docs/contributors/testing-overview.md

```diff
@@ -386,3 +386,18 @@ To run unit tests only, without the linter, use `npm run test-unit-php` instead.
 
 [snapshot testing]: https://jestjs.io/docs/en/snapshot-testing.html
 [update snapshots]: https://jestjs.io/docs/en/snapshot-testing.html#updating-snapshots
+
+## Native mobile testing
```

Comment:
Can you move it up just before the section about E2E testing? It's better when it's grouped together with the regular unit testing, it's nearly identical after all :)

Reply:
Sure, addressed with 3064350.